### ISSUE TITLE
Support text marks

### ISSUE THREAD

**Issue by arvind** [uwdata/lyra#227]
Lyra 2 should support Vega's `text` mark type. This will involve creating:
- [x] an primitive mark class.
- [x] a manipulators transform class
- [x] property inspector
- [ ] test cases for rules

**Comment by kadamwhite** [comment]
First three points ticked off in #281

**Comment by kadamwhite** [comment]
@arvind can you clarify what "test cases for rules" means -- functional test cases, or unit tests?

**Comment by arvind** [comment]
I meant functional test cases. I had started on some for [symbol marks](https://github.com/vega/lyra/blob/lyra2/test/rules.test.js) which I used to [produce PNGs](https://github.com/vega/lyra/blob/lyra2/test/resources/rules-fill.png). Producing PNGs affords visual inspection, but I'm not sure how robust it is against variations in server-side rendering (e.g., fonts). Another alternative is to generate the SVG strings and compare. Both of these, though, could come later and for now, we can just verify that rules produce the necessary properties on the model.
